php/ext/google/protobuf/protobuf.c

```diff
@@ -248,6 +250,23 @@ EnumDescriptorInternal* get_class_enumdesc(const char* klass) {
   }
 }
 
+void add_file(const char* file) {
+  printf("File registered: %s\n", file);
```

Comment:
@TeBoring printing each registered file could take a significant amount of time, is this really necessary?